website/docs/vectorextensions.md

```diff
@@ -13,14 +13,15 @@ If no value is set for a configuration, the toolset's default vector extension s
 | Value       | Description                                            |
 |-------------|--------------------------------------------------------|
 | Default     | Use the toolset's default vector extension settings.   |
-| AVX         | Use Advanced Vector Extensions.                        |
-| AVX2        | Use Advanced Vector Extensions 2.                      |
 | IA32        | Use Intel Architecture 32-bit                          |
 | SSE         | Use the basic SSE instruction set.                     |
 | SSE2        | Use the SSE2 instruction set.                          |
 | SSE3        | Use the SSE3 instruction set.                          |
 | SSSE3       | Use the SSSE3 instruction set.                         |
 | SSE4.1      | Use the SSE4.1 instruction set.                        |
+| SSE4.2      | Use the SSE4.2 instruction set.                        |
+| AVX         | Use Advanced Vector Extensions.                        |
+| AVX2        | Use Advanced Vector Extensions 2.                      |
```

Comment:
Reordering this to be based on release date instead of alphabetical should be done as a different PR - this PR is adding SSE4.2 and should only add SSE4.2.

Reply:
> this PR is adding SSE4.2 and should only add SSE4.2.